src/ds/bits.h

```diff
@@ -52,8 +52,6 @@ namespace snmalloc
       return (static_cast<T>(1)) << shift;
     }
 
-    static constexpr size_t ADDRESS_BITS = is64() ? 48 : 32;
```

Comment:
This has been moved into the PAL, but I wonder whether it's actually a mixture of PAL and AAL.  Should the PAL provide a maximum by select the minimum of its version and the AAL's version?

Reply:
I suppose so?  It's a little unfortunate in that if a platform ever widens its kernel-provided mapping of userland then an old build of snmalloc is going to break.